src/ontology/mondo-ingest.Makefile

```diff
@@ -177,6 +177,26 @@ metadata/mondo.sssom.config.yml:
 
 mappings: sssom $(ALL_MAPPINGS)
 
+# TODO: create script and add params here
+# TODO: dependencies: re-use for loop for ids?: ALL_MIRROR_SIGNTAURE_REPORTS=$(foreach n,$(ALL_COMPONENT_IDS), reports/component_signature-$(n).tsv)
+.PHONY: document-unmapped-terms
+document-unmapped-terms:
+	python3 $(SCRIPTSDIR)/mappings_unmapped_docs.py
+
+# TODO: dependencies: re-use for loop for ids?: ALL_MIRROR_SIGNTAURE_REPORTS=$(foreach n,$(ALL_COMPONENT_IDS), reports/component_signature-$(n).tsv)
+.PHONY: unmapped-terms-tables
+unmapped-terms-tables:
+	echo TODO
+
+$(REPORTDIR)/%_unmapped_terms.tsv $(REPORTDIR)/%_unmapped_terms_stats.tsv: $(REPORTDIR)/%_term_exclusions.txt (REPORTDIR)/mirror_signature-%.tsv $(TMPDIR)/mondo.sssom.tsv mirror/%.db
```

Comment:
Makefile goal for creating tables of unmapped terms, as well as statistical tables.

This wasn't specified in [the issue](https://github.com/monarch-initiative/mondo-ingest/issues/102), but I wonder if this is better: (i) a goal to create these tables for a given ontology, (ii) a phony goal to do so for all ontologies, (iii) a phony goal to update the docs for all ontologies at once.

Let me know if you also want (iv) a goal to update the documentation for a _single ontology_, but I don't have any intention of that right now.